indico/modules/events/layout/controllers/layout.py

```diff
@@ -74,12 +76,26 @@ def _process(self):
 
 
 class RHLayoutEdit(RHLayoutBase):
-    def _process(self):
+    def _process_GET(self):
         if self.event.type_ == EventType.conference:
             return self._process_conference()
         else:
             return self._process_lecture_meeting()
 
+    def _process_POST(self):
+        old_values = layout_settings.get_all(self.event)
+        ret = self._process_GET()
```

Comment:
I think the contents of `process_GET` could be extracted to a separate method which could be called here instead of `process_GET`

Reply:
Unless there is magic happening under the hood when calling `self.process_GET`, I think this adds unnecessary indirection. Is this what you would prefer?

```python
def __process(self):
    if self.event.type_ == EventType.conference:
        return self._process_conference()
    else:
        return self._process_lecture_meeting()

def _process_GET(self):
    self.__process()

def _process_POST(self):
    old_values = layout_settings.get_all(self.event)
    ret = self.__process()
    new_values = layout_settings.get_all(self.event)
```

Up to you. Let me know what you prefer.

Reply:
Yeah, there might no be anything happening under the hood but I still think this method should be used only internally and not overused by the classes implementing it.

Reply:
Done 👌